README.md

```diff
@@ -33,43 +38,42 @@ To generate an `index.json` template file, complete the following steps:
-* The file must include the `docsUrl` field inside the `apps` array, which points to the URL of your add-on documentation.
-
-The following information is optional:
-
-* Information about your organization:
-
-  * `name` and `description`
-  * `avatar` - Image or icon for your organization
-  * `contact` (recommended) - Support information displayed via the 'Support' button
-
-* Inside the `apps` array, an entry for each add-on you want to be shown in the index:
-
-  * `name` must match the add-on's repository name.
-  * `title` is the human-readable name of the repository.
-  * `description` is the short description of the add-on.
-  * `manifest` is the alternative name of the west manifest. Defaults to `west.yml`.
-  * `kind` is the type of add-on.
-  * `tags` are the tags that will be used to categorize the add-on.
-  * `license` is the license type name.
-  * `apps` is the global pattern to find directories containing add-ons.
-  * `releases` are the add-on versions.
-  * `avatar` - Image or icon for the add-on.
-  * `restricted` indicates if the access to the add-on or any of its dependencies is restricted to a specific audience.
-    This field requires providing an URL to a page that explains how to get access.
-    If this field is used, the add-on entry will be marked with a padlock on the index page.
-
-Most of the information provided in these entries will be displayed on the add-on index page.
-For more information about each entry, see `appMetadataSchema` in the `resources/schema.json` file.
+The `index.json` is compiled from the files in the `ncs-app-index/index` directory. To add your Add-on to the index you can either [create](#adding-your-organization-to-the-index) or modify existing file of your organization in `ncs-app-index/index` directory.
+
+### Adding your organization to the index
+
+To add your organization to the index, create an JSON of your own. The common practice is to name that file after the contributor's organization. The file is required to be located in `ncs-app-index/index` directory and follow the [`JSON schema`](./resources/schema.json).
+
+The following represents an example of the organization's index file:
+
+```
+{
+    "name": "Application maker",
+    "description": "A company that makes applications.",
+    "apps": [
+        {
+            "title": "Application #1",
+            "name": "application_1",
+            "description": "The first application",
+            "kind": "sample",
+            "tags": ["ble", "dfu"],
+            "avatar": "https://link-to-avatar
+            "releases": [
+                {
+                    "date": "2024-08-11T14:37:00Z",
+                    "name": "v2.0.1",
+                    "tag": "v2.0.1",
+                    "sdk": "v2.8.0"
+                }
+            ],
+            "docsUrl": "https://link-to-docs"
+        }
+    ]
+}
+
+```
+
+For more information about each entry, see `resources/schema.json` file.
 
 #### Schemas
```

Comment:
How about moving lines 78-80 to line 46 and removing this section?

Reply:
Good suggestion, thanks !